_sidebar.md

```diff
@@ -1,6 +1,7 @@
 * **Getting Started**
   * **Fixtures**
     * [Create a simple fixture](/fixture/simple-fixture.md)
+    * [Create a cms fixture](/fixture/cms-fixture.md)
```

Comment:
Can you create a CMS section explaining what to install? Or maybe a plugins section with the Bitbag cms plugin and a link to this page to explain how to create fixtures?